official/mnist/mnist.py

```diff
@@ -122,14 +131,14 @@ def model_fn(features, labels, mode, params):
         })
 
 
-def main(unused_argv):
-  data_format = FLAGS.data_format
+def main_with_model_fn(flags, unused_argv, model_function):
```

Comment:
Is it ok to leave this as part of `main`, or is there a reason for separating this function? Currently, it is only called in `main`.